src/client/scripts/room.js

```diff
@@ -119,10 +115,45 @@ Template.room.events({
         }
 
         Cards.update({ _id: this._id}, { $inc: {likes: 1} });
+    },
+
+    "keydown #editor": function(keyPressed){
+        var pressed = keyPressed.which || keyPressed.keyCode;
+
+        if(keyPressed.ctrlKey && pressed == 83){
```

Comment:
it appears you are trying to capture ctrl+s.  Look at this:
http://jsfiddle.net/29sVC/
It seems you can just do keyPressed.ctrlKey && keyPressed == *key value*
so you dont need the pressed variable thing you are doing
I think casting 's' to an int should give the ascii value and is better than the magic number IMO.